### PR TITLE
Change default path

### PR DESCRIPTION
**Pull request by danielkoster**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Default path is now /usr/local/composer/vendor/ due to OSX-restrictions.
```
Comment (sjadema) [inline CHANGELOG.md]:
```suggestion
- Default path is now /usr/local/lib/composer/vendor/ due to OSX-restrictions.
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 A plugin to install stable, [remote](https://packagist.org/) [Composer](https://getcomposer.org) packages in a global 
 vendor directory. The path to a global installed package is 
 `[global-vendor-dir]/[pretty-package-name]/[pretty-package-version]`; for example, `symfony/console:v5.3.1` might be 
-found under `/usr/lib/composer/vendor/symfony/console/v5.3.1`. It creates symlinks in the local vendor directory 
+found under `/usr/local/composer/vendor/symfony/console/v5.3.1`. It creates symlinks in the local vendor directory
```
Comment (sjadema) [inline README.md]:
```suggestion
found under `/usr/local/lib/composer/vendor/symfony/console/v5.3.1`. It creates symlinks in the local vendor directory 
```

File: README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ plugin global: `composer global require iwink/composer-global-installer`.
 You can configure the plugin using the `extra.global-installer` key in `composer.json` (and in the global 
 `$COMPOSER_HOME/composer.json`):
 
-- `path`: Path to global vendor directory. (default: `/usr/lib/composer/vendor/`)
+- `path`: Path to global vendor directory. (default: `/usr/local/composer/vendor/`)
```
Comment (sjadema) [inline README.md]:
```suggestion
- `path`: Path to global vendor directory. (default: `/usr/local/lib/composer/vendor/`)
```

File: res/schema.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
 		"path": {
 			"type": "string",
 			"description": "Path to install directory.",
-			"default": "/usr/lib/composer/vendor"
+			"default": "/usr/local/composer/vendor/"
```
Comment (sjadema) [inline res/schema.json]:
```suggestion
			"default": "/usr/local/lib/composer/vendor/"
```

**Review by sjadema — CHANGES_REQUESTED** [review]
Changed the default path to a more standarized path.

**Review by doekenorg — APPROVED** [review]

**Pull request merged**